client/my-sites/site-settings/settings-newsletter/main.tsx

```diff
@@ -112,12 +112,12 @@ const NewsletterSettingsForm = wrapSettingsForm( getFormSettings )( ( {
 		);
 	} );
 
-	const isSubscriptionOnCommentModuleInactive = useSelector( ( state ) => {
+	const shouldShowSubscriptionOnCommentModule = useSelector( ( state ) => {
 		const isJetpackSite = isJetpackSiteSelector( state, siteId, {
 			treatAtomicAsJetpackSite: true,
 		} );
 
-		return Boolean( isJetpackSite );
+		return ! isJetpackSite && 1 === lang_id;
```

Comment:
```suggestion
		return ! isJetpackSite;
```

Should we remove this check for `1 === lang_id` to enable the setting for all languages?

Reply:
You're right, I noticed I didn't saved the file 🤦